File: DSA/Arartik_Tyagi_590011633/DAY_1/Problem2_leetcode.java
class Solution {
    public int Problem2_leetcode(int[] nums) {
        int n = nums.length;
        int sum = 0;

        for (int num : nums) {
            sum += num;
        }

        int expectedSum = n * (n + 1) / 2;

        return expectedSum - sum;
    }
}

File: DSA/Kshitiz_590028542/Day1/Problem2_Leetcode.c
int missingNumber(int* nums, int numsSize) {

    int NaturalNumberSum = (numsSize * (numsSize + 1))/2;

    int sum = 0;
    for(int i = 0; i < numsSize; i++){
        sum += nums[i];
    }
    
return NaturalNumberSum - sum;
}